app/mailers/organization_account_requests_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class OrganizationAccountRequestsMailer < ApplicationMailer
+  def create_new_organization_account_request
+    @organization_name = params[:name]
+    @requester_name = params[:organization_requester_name]
+    @organization_phone = params[:phone_number]
+    @organization_email = params[:email]
+    # @country = params[:locations][:country]
```

Comment:
I'm having a problem accessing the locations param. could i get any help here?

Reply:
Hi @Meri-MG , the reason this is happening is sort of three-fold. 

1. In the form, you have the model as `Organization` but this is the name of the class and not an instance of a model. And as a result the params are coming back as containing `{ 'locations' =>  ... }` instead of `{ locations_attributes => ... }`. In order to get a `locations_attributes` key in the params, you'd need to pass an instance of the organization model, eg. `Organization.new`. 

2. Passing `organizations_params` to the mailer, there isn't a `locations` key because only `locations_attributes` were permitted

3. Switching `Organization.new` for `Organization` and then trying to access `params[:locations_attributes][:country]` etc in the mailer still wouldn't quite work because there's going to be an extra key to traverse to get to the value you want. The params will actually end up looking like `{ 'locations_attributes' => { '0' => { 'country' => ... } } ...}`. So to pick out the country value you'd end up doing something like `@country = params.dig('locations_attributes', '0', 'country')`

All that being said, switching to using a Contact model in the controller will mean not having to deal with this particular problem

Reply:
Oh, I see. thanks a lot for clarifying.